doc/changelog.rst

```diff
@@ -13,6 +13,7 @@ PyMongo 4.2 brings a number of improvements including:
   changes may be made before the final release.  See :ref:`automatic-queryable-client-side-encryption` for example usage.
 - Provisional (beta) support for :func:`pymongo.timeout` to apply a single timeout
   to an entire block of pymongo operations.
+- Added the ``timeoutMS`` URI and :class:`~pymongo.mongo_client.MongoClient` keyword argument.
```

Comment:
`... constructor keyword argument.` ?

Reply:
changed to `keyword argument to` which we use elsewhere.